src/CUDA2HIP_TENSOR_API_types.cpp

```diff
@@ -190,4 +203,8 @@ const std::map<llvm::StringRef, hipAPIversions> HIP_TENSOR_TYPE_NAME_VER_MAP {
   {"HIPTENSOR_OP_IDENTITY",                            {HIP_5070,       HIP_0,         HIP_0,       }},
   {"HIPTENSOR_OP_SQRT",                                {HIP_6020,       HIP_0,         HIP_0,       }},
   {"HIPTENSOR_OP_UNKNOWN",                             {HIP_5070,       HIP_0,         HIP_0,       }},
+  {"hiptensorAlgo_t",                                  {HIP_5070,       HIP_0,         HIP_0,       }},
+  {"HIPTENSOR_ALGO_ACTOR_CRITIC",                      {HIP_5070,       HIP_0,         HIP_0,       }},
```

Comment:
HIPIFY tools do not deal with HIP-specific APIs and Data types. Not being presented in the main CUDA2HIP mapping table, it will be presented nowhere: neither hipify-perl nor documentation.

Reply:
Got it, so we can skip these ones which don't map to another cuda API. Thanks!